DRAFT.md

```diff
@@ -275,6 +275,23 @@ propagates const and is allocator aware.
   (it could be an instance of a derived type). As a result `polymorphic` cannot
   forward comparison operators, hash or formatting to the owned object.
 
+#### Similarities and dissimilarities with variant
```

Comment:
Comparison with variant (?)

Reply:
It's ambiguous as were talking about comparison and indirect == variant could be a thing.